config/default-spinnaker-local.yml

```diff
@@ -121,6 +121,21 @@ providers:
       # A path to a plain text file containing the user's password
       passwordFile: ${SPINNAKER_DOCKER_PASSWORD_FILE}
 
+  openstack:
+    # For more information on configuring Openstack clusters, see
+    # TODO add link to docs when available
+
+    enabled: ${SPINNAKER_OPENSTACK_ENABLED:false}
+    primaryCredentials:
+      name: my-openstack-account
+      authUrl: ${SPINNAKER_OPENSTACK_AUTH_URL}
```

Comment:
The default values are empty. Here you are overriding them. Did you mean to have a default?